services/sidecar/rollup/services/disseminator/interface.go

```diff
@@ -19,18 +18,15 @@ type ForkChoiceResponse = engine.ForkChoiceResponse
 type BuildPayloadResponse = engine.ForkChoiceResponse
 
 type BatchBuilder interface {
-	Append(block derivation.DerivationBlock, header derivation.HeaderRef) error
-	LastAppended() types.BlockID
-	Build() ([]byte, error)
+	Enqueue(block *ethTypes.Block) error
```

Comment:
interface declaration usually does not contain method argument names

Reply:
can you cite a style guide for that? i thought its better to use so you can document the param usage